dbdimp.h

```diff
@@ -110,22 +124,22 @@
  */
 
 /* Use mysql_options with MYSQL_OPT_SSL_VERIFY_SERVER_CERT */
-#if ((MYSQL_VERSION_ID >= 50023 && MYSQL_VERSION_ID < 50100) || MYSQL_VERSION_ID >= 50111) && (MYSQL_VERSION_ID < 80000 || defined(MARIADB_BASE_VERSION))
+#ifdef CLIENT_SSL_VERIFY_SERVER_CERT
```

Comment:
The verification is still in place:
https://github.com/mysql/mysql-server/blob/8.0/sql-common/client.cc#L3715

Even if MYSQL_OPT_SSL_VERIFY_SERVER_CERT was renoved, you can still pass CLIENT_SSL_VERIFY_SERVER_CERT flag to mysql_real_connect (last parameter).